Ff154 css typed OM cleanup (#44945)
mdn/content: files/en-us/web/api/stylepropertymap/set/index.md

{{APIRef("CSS Typed Object Model API")}}

> The **`set()`** method of the {{domxref("StylePropertyMap")}} interface changes the CSS declaration with the given property.

Reviewer: ```suggestion
The **`set()`** method of the {{domxref("StylePropertyMap")}} interface changes the CSS declaration using the given property.
```

## Examples

### Basic usage

> This example sets the {{cssxref('padding-top')}} property, with the given value, within the element's [style attribute](/en-US/docs/Web/HTML/Reference/Global_attributes/style).

Reviewer: ```suggestion
This example sets the specified value for the {{cssxref('padding-top')}} property within the element's [style attribute](/en-US/docs/Web/HTML/Reference/Global_attributes/style).
```
